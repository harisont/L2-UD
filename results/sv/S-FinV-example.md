## Sentence 1958:
L1 sentence | L2 sentence
--- | ---
**Därför** **tycker** **jag** **om** **havet** **.** | **Därför** **jag** **tycker** **om** **havet** **.**


## Sentence 1943:
L1 sentence | L2 sentence
--- | ---
**Tyvärr** **har** **någonting** **hänt** **som** **gör** **att** **jag** **inte** **kan** **gå** **på** **kursen** **och** **jag** **önskar** **att** **få** **pengarna** **tillbaka** **.** | **Tyvärr** **någonting** **har** **hänt** **som** **gör** **att** **jag** **inte** **kan** **gå** **på** **kursen** **och** **jag** **önskar** **att** **få** **pengarna** **tillbaka** **.**


## Sentence 1965:
L1 sentence | L2 sentence
--- | ---
**Där** **bodde** **jag** **i** **Göteborg** **med** **min** **mamma** **och** **hennes** **hundar** **i** **hennes** **hus** **.** | **Där** **jag** **bodde** **i** **Göteborg** **med** **min** **mamma** **och** **hennes** **hundar** **i** **hennes** **hus** **.**


## Sentence 1936:
L1 sentence | L2 sentence
--- | ---
Ibland **kan** **vi** **titta** och lyssna på hur det funkar . | Ibland **vi** **kan** **titta** och lyssna på hur det funkar .


## Sentence 1139:
L1 sentence | L2 sentence
--- | ---
**Därför** har **man** **kommit** med förslaget att ha en kurs angående arbetslivet i gymnasiet **.** | **Därför** **man** **kommit** med förslaget att ha en kurs angående arbetslivet i gymnasiet **.**


## Sentence 1942:
L1 sentence | L2 sentence
--- | ---
Lyckligtvis kan du spela piano , och **där** kan **du** **lära** känna nya vänner . | Lyckligtvis kan du spela piano , och **där** **du** kan **lära** känna nya vänner .


## Sentence 1976:
L1 sentence | L2 sentence
--- | ---
**Ibland** **brukar** **man** **säga** **att** **kärleken** **har** **ingen** **gräns** **och** **det** **är** **sant** **.** | **Ibland** **man** **brukar** **säga** **att** **kärleken** **har** **ingen** **gräns** **och** **det** **är** **sant** **.**


## Sentence 1939:
L1 sentence | L2 sentence
--- | ---
Men nu **är** **jag** inte **intresserad** **av** **den** **längre** . | Men nu **jag** **är** inte **intresserad** **av** **den** **längre** .


