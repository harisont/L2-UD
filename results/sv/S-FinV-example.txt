1958
1943
1950
1965
1936
1139
1942
1976
1939
