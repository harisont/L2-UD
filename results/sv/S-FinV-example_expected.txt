(1907)
1935..1937
1939..1940
1942
1943
1945?
WIP

to check:
- 1935-1986